tasks/update-code/copy_zipped.yml

```diff
@@ -0,0 +1,29 @@
+---
+
+- name: ANSISTRANO | SCP_ZIP | Create release folder
+  file:
+    state: directory
+    path: "{{ ansistrano_release_path.stdout }}"
+
+- name: ANSISTRANO | SCP_ZIP | Set archived file
+  set_fact:
+    ansistrano_archived_file: "{{ ansistrano_release_path.stdout }}/data.gz"
+    work_file: "/tmp/ansistrano.gz"
+
+- name: ANSISTRANO | SCP_ZIP | Zip source file
+  command: tar czfh {{work_file}} {% for fn in ansistrano_zip_exclude %} --exclude {{fn}} {% endfor %} -C {{ ansistrano_deploy_from }} .
+  delegate_to: localhost
+
+- name: ANSISTRANO | SCP_ZIP | Upload files
+  copy:
+    src: "{{ work_file }}"
+    dest: "{{ ansistrano_archived_file }}"
+
+- name: ANSISTRANO | SCP_ZIP | Remove temporal file
+  file:
+    path: '{{work_file}}'
```

Comment:
CS here

Reply:
Sorry I don't understand the mean of the word 'CS'.
client side? is there any better method than delegate_to?

Reply:
Coding standards, it should be 

```
    path: '{{ work_file }}'
```